warehouse/models/mart/transit_database/_mart_transit_database.yml

```diff
@@ -176,7 +176,7 @@ models:
           **Note: This field is manually entered in the Transit Database,
           so it may be subject to some gaps in coverage and it is not updated automatically.
           Data in this field may not align with validations data from the
-          GTFS pipeline, for example in `views.validation_fact_daily_feed_notices`,
+          GTFS pipeline, for example in `mart_gtfs.fct_daily_rt_feed_files`,
```

Comment:
The equivalent table to the one previously referenced would be `mart_gtfs_quality.fct_daily_schedule_feed_validation_notices`

Reply:
Resolved in [e6f2773](https://github.com/cal-itp/data-infra/pull/2602/commits/e6f2773037116a38d5e1b6783fa7c63729318178)